### DISCUSSION TITLE
Nuget version 5.0.0-ci552 resource “themes/materialdesigntheme.defaults.xaml” not found

### DISCUSSION THREAD

**Discussion by Broderick890** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3472] [Q&A] [answered]
<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />

IOException: resource “themes/materialdesigntheme.defaults.xaml” not found 。
Please help!

**Comment by Keboo** [comment, marked as answer]
This is an intentional change for 5.0.0. You need to change that to either using `MaterialDesign2.Defaults.xaml` or `MaterialDesign3.Defaults.xaml`. See the pinned announcement for more details. https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/discussions/3466

**Comment by Broderick890** [comment]
Thank you!
